### PR TITLE
Feature/additional styling

### PR DESCRIPTION
**Pull request by hborrelli1**
## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Styling- no new features
- [X] Refactoring- no new features

## Detailed Description
- Added styling to the homepage and movie details page. 
- Had to refactor somethings slightly in order to style consitently. 

## Files modified:
- src/components/App/App.scss
- src/components/Login/Login.js
- src/components/Login/Login.scss
- src/components/MovieCard/MovieCard.scss
- src/components/MovieContainer/MovieContainer.scss
- src/components/MovieDetails/MovieDetails.js
- src/components/MovieDetails/MovieDetails.scss

### CODE REVIEW

File: src/components/MovieDetails/MovieDetails.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class MovieDetails extends Component {
 
   displayUserRatingConditional = () => {
     if(this.props.userInfo.id){
-      return `Your rating: ${this.showUserRating()}`
+      return this.showUserRating();
```
Comment (hborrelli1) [inline src/components/MovieDetails/MovieDetails.js:47]: I broke this text out from the function and added to the component so I could add in a span tag around the user rating. See changes below.

File: src/components/MovieDetails/MovieDetails.scss [hunk 2]
```diff
@@ -24,11 +44,12 @@ a {
   flex-flow: row wrap;
   align-items: center;
   font-size: 1em;
-  width: 30%;
+  // width: 30%;
```
Comment (CLPena) [inline src/components/MovieDetails/MovieDetails.scss:47]: Is this commented out? Not a huge deal, but might want to just delete it if so.

File: src/components/MovieDetails/MovieDetails.js [hunk 3]
```diff
@@ -102,24 +102,24 @@ class MovieDetails extends Component {
...
+          </section>
+          <section className="movie-details">
+            <section>
+              <h1>{title}</h1>
+              <h3 className="average-rating">Average rating: <span>{average_rating && average_rating.toFixed(1)}/10</span></h3>
+
+              {this.props.userInfo.id && <h3 className="user-rating">Your rating: <span>{this.displayUserRatingConditional()}</span></h3>}
+
+              <h3>Release date: <span>{release_date}</span></h3>
+            </section>
```
Comment (hborrelli1) [inline src/components/MovieDetails/MovieDetails.js]: These are the changes from above.

File: src/components/MovieDetails/MovieDetails.js (hunk 1 above)
Comment (CLPena) [reply src/components/MovieDetails/MovieDetails.js:47]: Great idea!

**Comment by CLPena** [comment]
Looks great! Super clean and no redundancies that I can spot.

File: src/components/MovieDetails/MovieDetails.js (hunk 3 above)
Comment (megan-venetianer) [reply src/components/MovieDetails/MovieDetails.js]: Thanks for clarifying! Looks great

**Pull request merged**